File: test_1/compare.py
import time

max = 100000
temperature = 25.2

t1 = time.time()
for i in range(max):
    for j in range(max):
        temperature = i*j+temperature

t2 = time.time()

print("python ", t2-t1)

# pour max = 100_000
# FORTRAN 23.529 secondes
# python 430.110 secondes
# C++ 21 secondes

# avec calcul :
# FORTRAN 40.601 secondes
# python 2084.728 secondes
# C++ 33 secondes

File: test_1/compare.cpp
#include <ctime>
#include <iostream>

int main(){
    int max = 100000;
    double temperature = 25.2;
    std::time_t t1 = std::time(nullptr);

    for(int i=0;i<max;i++){
        for (int j=0;j<max;j++){
            temperature = i*j+temperature;
        }
    }

    std::time_t t2 = std::time(nullptr);

    std::cout<<"Temps C++ : "<<t2-t1<<std::endl;




    return 0;
}